<?xml version="1.0"?>

<launch>

    <node name="em_point_cloud" pkg="em_follow_me" type="em_point_cloud.py" output="screen" cwd="node" >
        <param name="use_color" value="false" />
        <param name="output_flag" value="true" />
        <param name="heightThreshold" value="0.05" />
    </node>

</launch>
